### PR TITLE
updated advanced git challenge

### PR DESCRIPTION
**Pull request by achandra16**

### CODE REVIEW

File: challenges/21_advanced_git/advanced_git_challenge.md [hunk 1]
```diff
@@ -191,11 +103,9 @@ In order to move forward, you will need to edit the file so that only the text y
...
 
-3. Once you have modified the code in your text editor, you will be able to finish making this commit by using `git add`.
-<img src="images/git-merge-message.png" width="500">
+3. Once you have modified the code in your text editor, you will be able to finish making this commit. 
 
-4. You can either write your own commit message, or accept the default.
-<img src="images/merge-commit-message.png" width="500">
+You can see another example of creating a merge conflict [here.](http://swcarpentry.github.io/git-novice/09-conflict/index.html)
 
 ## Advanced options
```
Comment (pab2163) [inline challenges/21_advanced_git/advanced_git_challenge.md:110]: I wonder whether we should leave the advanced options out for now -- might get confusing for students?

**Review by pab2163 — COMMENTED** [review]
@achandra16 and I have been discussing the changes @ashiq-r31 I discussed (sorry for forgetting to update everyone) for forking a separate repo here, and setting up the challenge so that the merge conflicts are 1) local and b) there are unit tests set up that will only pass once changes from both branches are incorporated

**Comment by ashiq-r31** [comment]
^ Ironically you need to resolve conflicts in this branch

**Comment by achandra16** [comment]
> ^ Ironically you need to resolve conflicts in this branch

:(

File: challenges/21_advanced_git/advanced_git_challenge.md [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
 
 While [some people](https://en.wikipedia.org/wiki/Editor_war#:~:text=vi%20is%20a%20smaller%20and,vim%20is%20almost%20as%20fast.&text=Emacs%20uses%20metakey%20chords.) love these text editors, they can be a bit cumbersome to learn especially when you are new to using terminal.
 
-Luckily, we can configure our Git session to use whatever text editor we prefer. Here are some options for other text editors you have used in class so far:
-- Atom `git config --global core.editor "atom --wait"`
-- Sublime (on a Mac) `git config --global core.editor "/Applications/Sublime\ Text.app/Contents/SharedSupport/bin/subl -n -w"`
-- Sublime (on a 64 bit PC) `git config --global core.editor "'c:/program files/sublime text 3/sublime_text.exe' -w"`
+Luckily, we can configure our Git session to use whatever text editor we prefer. Here is how to do this for VS Code:
+-`git config --global core.editor "code --wait"`
```
[the remark this answers is not in the document]
Comment (ashiq-r31) [reply challenges/21_advanced_git/advanced_git_challenge.md:21]: I agree with @pab2163, they may run into issues here but I also think setting up their default code editor might be necessary for this class to go smoothly

**Pull request merged**